gnosis/safe-react
# Fix: use address book name when loading a Safe

## What it solves
Resolves #2900

## How this PR fixes it
Grabs a Safe name from the Address Book if available.

## How to test it
* Add an owned Safe address to the address book (but don't add the Safe itself)
* Try adding this Safe from the sidebar
* Name and address should be pre-filled

## Screenshot
<img width="688" alt="Screenshot 2021-10-27 at 10 49 44" src="https://user-images.githubusercontent.com/381895/139032529-19928383-cf94-4caa-9493-eddd830f0024.png">
